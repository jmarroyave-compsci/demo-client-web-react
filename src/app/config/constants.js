import _package_ from 'package.json';

const Constants = {
	DEBUG : process.env.NEXT_PUBLIC__DEBUG,
	LOCAL : process.env.NEXT_PUBLIC__LOCAL,
	APP : {
		BASE_PATH: process.env.NEXT_PUBLIC__SITE_URL,
		SITE_URL: process.env.NEXT_PUBLIC__SITE_URL,
		VERSION : _package_.version,
		TITLE: process.env.NEXT_PUBLIC__TITLE,
		PAGE_TITLE: process.env.NEXT_PUBLIC__PAGE_TITLE
	},
	SERVERS: {
		DATA: process.env.NEXT_PUBLIC__DATA, 
		GRAPHQL :  process.env.NEXT_PUBLIC__DATA_GRAPHQL,
	},
	DEFAULTS : {
		PAGING_PAGE_SIZE: 10,
	},
	PLUGINS : {
		GOOGLE: {
			ANALYTICS: process.env.NEXT_PUBLIC_GOOGLE_ANALYTICS,
			MAPS: process.env.NEXT_PUBLIC__GOOGLE_MAPS_KEY,
		},
	},
	FEATURES : {
		SEARCH : (process.env.NEXT_PUBLIC__FEATURE_SEARCH == "true"),
	},
}

export default Constants;