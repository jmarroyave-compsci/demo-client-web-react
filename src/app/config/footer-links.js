import _links from './links';

const links = [
  { "name" : "jmarroyave", items: [
    { "name": "blog", "link" : _links.blog },
  ] },
  { "name" : "labs", items: [
    { "name": "stackoverflow", "link" : _links.labs_trends },
    { "name": "mlb", "link" : _links.labs_mlb },
    { "name": "search", "link" : _links.labs_search },
  ] },
  { "name" : "knowledge base", items: [
    { "name": "android", "link" : _links.kb_android },
  ] },
];

export default links;