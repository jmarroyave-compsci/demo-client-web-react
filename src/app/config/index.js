import _package_ from 'package.json';

const Constants = {
	TEST: (process.env.TESTING === "true") ? true : false,
	DEBUG : process.env.NEXT_PUBLIC__DEBUG,
	LOCAL : process.env.NEXT_PUBLIC__LOCAL,
	APP : {
		BASE_PATH: process.env.NEXT_PUBLIC__BASE_PATH,
		SITE_URL: process.env.NEXT_PUBLIC__SITE_URL,
		VERSION : _package_.version,
		TITLE: process.env.NEXT_PUBLIC__TITLE,
		DESCRIPTION: process.env.NEXT_PUBLIC__DESCRIPTION,
		PAGE_TITLE: process.env.NEXT_PUBLIC__PAGE_TITLE,
		HOME_URL: `${process.env.NEXT_PUBLIC__SITE_URL}${process.env.NEXT_PUBLIC__BASE_PATH}`,
	},
	SERVERS: {
		DATA: process.env.NEXT_PUBLIC__SERVER, 
		WEB_SOCKETS: process.env.NEXT_PUBLIC__SERVER, 
		GRAPHQL :  `${process.env.NEXT_PUBLIC__SERVER}/${process.env.NEXT_PUBLIC__SERVER_API_VERSION}/graphql`,
	},
	DEFAULTS : {
		PAGING_PAGE_SIZE: 10,
		IMAGE_PLACEHOLDER: "/img/ph.jpg",
	},
	PLUGINS : {
		GOOGLE: {
			ANALYTICS: process.env.NEXT_PUBLIC_GOOGLE_ANALYTICS,
			MAPS: process.env.NEXT_PUBLIC__GOOGLE_MAPS_KEY,
		},
	},
	FEATURES : {
		SEARCH : (process.env.NEXT_PUBLIC__FEATURE_SEARCH == "true"),
	},
	SOCIAL_NETWORKS: {
		CARD : {
			TITLE : `${process.env.NEXT_PUBLIC__TITLE} v.${_package_.version}`,
			THUMBNAIL_TWITTER: `${process.env.NEXT_PUBLIC__BASE_PATH}${process.env.NEXT_PUBLIC__CARD_THUMBNAIL_TWITTER}`,
		},
		TWITTER: process.env.NEXT_PUBLIC__SOCIAL_NETWORKS_TWITTER,
	}
}

export default Constants;