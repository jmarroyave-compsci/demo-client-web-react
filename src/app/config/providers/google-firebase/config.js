const config = {
  apiKey: "AIzaSyDaNheMMUtsqx8AjiJAWDcfXwAEBYLJUVY",
  authDomain: "labs-f885b.firebaseapp.com",
  projectId: "labs-f885b",
  //projectId: "labs-f885",
  storageBucket: "labs-f885b.appspot.com",
  messagingSenderId: "606766444808",
  appId: "1:606766444808:web:41eec12c129d582727bcbd",
  measurementId: "G-TNBYCW57TS"
};

export default config;