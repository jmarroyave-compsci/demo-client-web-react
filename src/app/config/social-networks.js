const links = [
  /*
	{enable: true, url: 'https://plus.google.com/112618720945759819017', icon: 'google-plus', color: '#d34836'},
	{enable: true, url: 'https://twitter.com/jmarroyave', icon: 'twitter', color: '#00aced'},
	*/
  { enable: true, url: 'https://facebook.com/jmarroyave', icon: 'facebook', color: '#3b5998' },
  { enable: true, url: 'https://www.linkedin.com/in/jmarroyave', icon: 'linkedin', color: '#0077B5' },
  { enable: true, url: 'https://github.com/jmarroyave', icon: 'github', color: '#000' },
  { enable: true, url: 'mailto:jmarroyave.compsci@gmail.com', icon: 'envelope-o', color: '#000' },
];


export default links;