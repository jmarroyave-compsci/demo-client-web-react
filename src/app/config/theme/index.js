import React from 'react';
import { createTheme } from '@mui/material/styles';
import { red } from '@mui/material/colors';

// Create a theme instance.
export const getTheme = ( mode ) => {
  return createTheme({
    palette: {
      mode: mode,
      primary: {
        main: '#556cd6',
      },
      secondary: {
        main: '#19857b',
      },
      error: {
        main: red.A400,
      },
    },
  });
}