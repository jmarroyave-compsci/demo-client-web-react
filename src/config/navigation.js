
class Navigation {
  constructor(){
    this.drawer = {
      show: false,
    };

    this.options = [
      {
        showInMenu: true, 
        showInDrawer: true, 
        name: 'Home', 
        icon: 'home', 
        route: '/', 
      },
      {
        showInMenu: true, 
        showInDrawer: true, 
        separator: true,
      },
      {
        showInMenu: true, 
        showInDrawer: true, 
        name: 'Movies', 
        icon: 'movie', 
        route: '/movies', 
      },
        {
          showInMenu: false, 
          showInDrawer: false, 
          name: 'Movie', 
          icon: 'movie', 
          route: '/movie/:id/:name', 
        },
      {
        showInMenu: true, 
        showInDrawer: true, 
        name: 'Podcasts', 
        icon: 'podcasts', 
        route: '/podcasts', 
      },
        {
          showInMenu: false, 
          showInDrawer: false, 
          name: 'Podcast', 
          icon: 'podcast', 
          route: '/podcast/:id/:name', 
        },
      {
        showInMenu: true, 
        showInDrawer: true, 
        separator: true,
      },
      {
        showInMenu: true, 
        showInDrawer: true, 
        name: 'People', 
        icon: 'people', 
        route: '/people', 
      },
        {
          showInMenu: false, 
          showInDrawer: false, 
          name: 'Person', 
          icon: 'person', 
          route: '/person/:id/:name', 
        },
      {
        showInMenu: false, 
        showInDrawer: false, 
        name: 'Awards', 
        route: '/stories/awards/:entity?/:year?/:page?', 
      },
      {
        showInMenu: false, 
        showInDrawer: false, 
        name: 'Stories', 
        route: '/stories', 
      },
      {
        showInMenu: false, 
        showInDrawer: false, 
        name: 'PodcastMusic', 
        route: '/podcasts/stories/music/:page?', 
      },
      {
        showInMenu: true, 
        showInDrawer: true, 
        separator: true,
      },
      {
        showInMenu: true, 
        showInDrawer: true, 
        name: 'About', 
        icon: 'info', 
        route: '/about', 
      },
      {
        showInMenu: false, 
        showInDrawer: false, 
        name: 'Search', 
        icon: 'search', 
        route: '/search/:qry/:page?', 
      },
    ];
  }
}



const nav = new Navigation();
export default nav;