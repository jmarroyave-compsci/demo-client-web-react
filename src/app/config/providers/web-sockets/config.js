import config from 'app/config';

const CONF = {
  SERVER: config.SERVERS.WEB_SOCKETS
};

export default CONF;