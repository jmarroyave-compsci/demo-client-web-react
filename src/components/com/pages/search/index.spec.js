import React from 'react';
import {render, fireEvent, waitFor, screen} from '@testing-library/react'
import '@testing-library/jest-dom'

import { itMustHaveNoErrors, TestProvider } from "tests/jest/shared"

import itemState from './automata/tests/item.json'
import TestComponent from '.';
import config from './.config';

describe(`${config.automata.name.toUpperCase()}`, () => {
  beforeEach(() => { 
  }); 
 
  it('compare Page against snapshot', () => {
    const renderer = render(
      <TestProvider query="search" namespace={config.automata.name} state={itemState}>
        <TestComponent render="page"/>
      </TestProvider>
    )

    itMustHaveNoErrors(renderer)
    expect(renderer.container).toMatchSnapshot();
  });
});