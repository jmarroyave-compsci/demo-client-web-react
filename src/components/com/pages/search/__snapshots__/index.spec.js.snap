// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SEARCH compare against snapshot 1`] = `
<div>
  <div
    class="css-nen11g-MuiStack-root"
  >
    <div
      class="css-1exnf3f-MuiStack-root"
    >
      <div
        class="MuiGrid-root page-module MuiGrid-container"
      >
        <div
          class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-12 MuiGrid-grid-md-9"
        >
          <div
            class="css-1tltgwc"
          >
            <div
              class="css-fxtdit"
            >
              <div
                class="css-3gbbd7"
              >
                <div>
                  search results for: 
                  <span
                    class="css-in3yi3"
                  >
                    transformers
                  </span>
                </div>
              </div>
              <div
                class="css-1i68010"
              >
                <h3
                  class="css-1wsh0w7"
                >
                  Filters
                </h3>
                <div
                  class="MuiFormGroup-root css-dmmspl-MuiFormGroup-root"
                >
                  <div
                    class="MuiGrid-root MuiGrid-container"
                  >
                    <div
                      class="MuiGrid-root MuiGrid-item"
                    >
                      <label
                        class="MuiFormControlLabel-root MuiFormControlLabel-labelPlacementTop css-1hpqosy-MuiFormControlLabel-root"
                      >
                        <span
                          class="MuiCheckbox-root MuiCheckbox-colorPrimary MuiButtonBase-root MuiCheckbox-root MuiCheckbox-colorPrimary PrivateSwitchBase-root Mui-checked css-12wnr2w-MuiButtonBase-root-MuiCheckbox-root"
                        >
                          <input
                            checked=""
                            class="PrivateSwitchBase-input css-1m9pwf3"
                            data-indeterminate="false"
                            name="person"
                            type="checkbox"
                          />
                          <svg
                            aria-hidden="true"
                            class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-i4bv87-MuiSvgIcon-root"
                            data-testid="CheckBoxIcon"
                            focusable="false"
                            viewBox="0 0 24 24"
                          >
                            <path
                              d="M19 3H5c-1.11 0-2 .9-2 2v14c0 1.1.89 2 2 2h14c1.11 0 2-.9 2-2V5c0-1.1-.89-2-2-2zm-9 14l-5-5 1.41-1.41L10 14.17l7.59-7.59L19 8l-9 9z"
                            />
                          </svg>
                          <span
                            class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
                          />
                        </span>
                        <span
                          class="MuiTypography-root MuiTypography-body1 MuiFormControlLabel-label css-ahj2mt-MuiTypography-root"
                        >
                          people
                        </span>
                      </label>
                    </div>
                    <div
                      class="MuiGrid-root MuiGrid-item"
                    >
                      <label
                        class="MuiFormControlLabel-root MuiFormControlLabel-labelPlacementTop css-1hpqosy-MuiFormControlLabel-root"
                      >
                        <span
                          class="MuiCheckbox-root MuiCheckbox-colorPrimary MuiButtonBase-root MuiCheckbox-root MuiCheckbox-colorPrimary PrivateSwitchBase-root Mui-checked css-12wnr2w-MuiButtonBase-root-MuiCheckbox-root"
                        >
                          <input
                            checked=""
                            class="PrivateSwitchBase-input css-1m9pwf3"
                            data-indeterminate="false"
                            name="movie"
                            type="checkbox"
                          />
                          <svg
                            aria-hidden="true"
                            class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-i4bv87-MuiSvgIcon-root"
                            data-testid="CheckBoxIcon"
                            focusable="false"
                            viewBox="0 0 24 24"
                          >
                            <path
                              d="M19 3H5c-1.11 0-2 .9-2 2v14c0 1.1.89 2 2 2h14c1.11 0 2-.9 2-2V5c0-1.1-.89-2-2-2zm-9 14l-5-5 1.41-1.41L10 14.17l7.59-7.59L19 8l-9 9z"
                            />
                          </svg>
                          <span
                            class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
                          />
                        </span>
                        <span
                          class="MuiTypography-root MuiTypography-body1 MuiFormControlLabel-label css-ahj2mt-MuiTypography-root"
                        >
                          movies
                        </span>
                      </label>
                    </div>
                    <div
                      class="MuiGrid-root MuiGrid-item"
                    >
                      <label
                        class="MuiFormControlLabel-root MuiFormControlLabel-labelPlacementTop css-1hpqosy-MuiFormControlLabel-root"
                      >
                        <span
                          class="MuiCheckbox-root MuiCheckbox-colorPrimary MuiButtonBase-root MuiCheckbox-root MuiCheckbox-colorPrimary PrivateSwitchBase-root Mui-checked css-12wnr2w-MuiButtonBase-root-MuiCheckbox-root"
                        >
                          <input
                            checked=""
                            class="PrivateSwitchBase-input css-1m9pwf3"
                            data-indeterminate="false"
                            name="tv_show"
                            type="checkbox"
                          />
                          <svg
                            aria-hidden="true"
                            class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-i4bv87-MuiSvgIcon-root"
                            data-testid="CheckBoxIcon"
                            focusable="false"
                            viewBox="0 0 24 24"
                          >
                            <path
                              d="M19 3H5c-1.11 0-2 .9-2 2v14c0 1.1.89 2 2 2h14c1.11 0 2-.9 2-2V5c0-1.1-.89-2-2-2zm-9 14l-5-5 1.41-1.41L10 14.17l7.59-7.59L19 8l-9 9z"
                            />
                          </svg>
                          <span
                            class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
                          />
                        </span>
                        <span
                          class="MuiTypography-root MuiTypography-body1 MuiFormControlLabel-label css-ahj2mt-MuiTypography-root"
                        >
                          tv shows
                        </span>
                      </label>
                    </div>
                    <div
                      class="MuiGrid-root MuiGrid-item"
                    >
                      <label
                        class="MuiFormControlLabel-root MuiFormControlLabel-labelPlacementTop css-1hpqosy-MuiFormControlLabel-root"
                      >
                        <span
                          class="MuiCheckbox-root MuiCheckbox-colorPrimary MuiButtonBase-root MuiCheckbox-root MuiCheckbox-colorPrimary PrivateSwitchBase-root Mui-checked css-12wnr2w-MuiButtonBase-root-MuiCheckbox-root"
                        >
                          <input
                            checked=""
                            class="PrivateSwitchBase-input css-1m9pwf3"
                            data-indeterminate="false"
                            name="video_game"
                            type="checkbox"
                          />
                          <svg
                            aria-hidden="true"
                            class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-i4bv87-MuiSvgIcon-root"
                            data-testid="CheckBoxIcon"
                            focusable="false"
                            viewBox="0 0 24 24"
                          >
                            <path
                              d="M19 3H5c-1.11 0-2 .9-2 2v14c0 1.1.89 2 2 2h14c1.11 0 2-.9 2-2V5c0-1.1-.89-2-2-2zm-9 14l-5-5 1.41-1.41L10 14.17l7.59-7.59L19 8l-9 9z"
                            />
                          </svg>
                          <span
                            class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
                          />
                        </span>
                        <span
                          class="MuiTypography-root MuiTypography-body1 MuiFormControlLabel-label css-ahj2mt-MuiTypography-root"
                        >
                          video games
                        </span>
                      </label>
                    </div>
                    <div
                      class="MuiGrid-root MuiGrid-item"
                    >
                      <label
                        class="MuiFormControlLabel-root MuiFormControlLabel-labelPlacementTop css-1hpqosy-MuiFormControlLabel-root"
                      >
                        <span
                          class="MuiCheckbox-root MuiCheckbox-colorPrimary MuiButtonBase-root MuiCheckbox-root MuiCheckbox-colorPrimary PrivateSwitchBase-root Mui-checked css-12wnr2w-MuiButtonBase-root-MuiCheckbox-root"
                        >
                          <input
                            checked=""
                            class="PrivateSwitchBase-input css-1m9pwf3"
                            data-indeterminate="false"
                            name="podcast"
                            type="checkbox"
                          />
                          <svg
                            aria-hidden="true"
                            class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-i4bv87-MuiSvgIcon-root"
                            data-testid="CheckBoxIcon"
                            focusable="false"
                            viewBox="0 0 24 24"
                          >
                            <path
                              d="M19 3H5c-1.11 0-2 .9-2 2v14c0 1.1.89 2 2 2h14c1.11 0 2-.9 2-2V5c0-1.1-.89-2-2-2zm-9 14l-5-5 1.41-1.41L10 14.17l7.59-7.59L19 8l-9 9z"
                            />
                          </svg>
                          <span
                            class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
                          />
                        </span>
                        <span
                          class="MuiTypography-root MuiTypography-body1 MuiFormControlLabel-label css-ahj2mt-MuiTypography-root"
                        >
                          podcasts
                        </span>
                      </label>
                    </div>
                    <div
                      class="MuiGrid-root MuiGrid-item"
                    >
                      <label
                        class="MuiFormControlLabel-root MuiFormControlLabel-labelPlacementTop css-1hpqosy-MuiFormControlLabel-root"
                      >
                        <span
                          class="MuiCheckbox-root MuiCheckbox-colorPrimary MuiButtonBase-root MuiCheckbox-root MuiCheckbox-colorPrimary PrivateSwitchBase-root Mui-checked css-12wnr2w-MuiButtonBase-root-MuiCheckbox-root"
                        >
                          <input
                            checked=""
                            class="PrivateSwitchBase-input css-1m9pwf3"
                            data-indeterminate="false"
                            name="festival"
                            type="checkbox"
                          />
                          <svg
                            aria-hidden="true"
                            class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-i4bv87-MuiSvgIcon-root"
                            data-testid="CheckBoxIcon"
                            focusable="false"
                            viewBox="0 0 24 24"
                          >
                            <path
                              d="M19 3H5c-1.11 0-2 .9-2 2v14c0 1.1.89 2 2 2h14c1.11 0 2-.9 2-2V5c0-1.1-.89-2-2-2zm-9 14l-5-5 1.41-1.41L10 14.17l7.59-7.59L19 8l-9 9z"
                            />
                          </svg>
                          <span
                            class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
                          />
                        </span>
                        <span
                          class="MuiTypography-root MuiTypography-body1 MuiFormControlLabel-label css-ahj2mt-MuiTypography-root"
                        >
                          movie festivals
                        </span>
                      </label>
                    </div>
                  </div>
                </div>
              </div>
              <div
                class="css-0"
              >
                <div
                  class="css-7ry9k1"
                >
                  <div
                    class="css-1p5q5e5-MuiStack-root"
                  >
                    <div
                      class="css-1p5q5e5-MuiStack-root"
                    >
                      <div
                        class="css-0"
                      >
                        <div
                          class="css-1fe9i7b"
                        >
                          Video Game
                        </div>
                        <h3
                          class="css-8kn4zf"
                        >
                          <span
                            class="css-pt399b"
                          >
                            transformers
                          </span>
                        </h3>
                        <div
                          class="css-1gqgroh"
                        >
                          released: 2004-01-01T05:00:00.000Z
                        </div>
                      </div>
                      <hr
                        class="MuiDivider-root MuiDivider-fullWidth MuiDivider-flexItem css-yk10cl-MuiDivider-root"
                      />
                      <div
                        class="css-0"
                      >
                        <div
                          class="css-1fe9i7b"
                        >
                          Movie
                        </div>
                        <h3
                          class="css-8kn4zf"
                        >
                          <span
                            class="css-pt399b"
                          >
                            transformers
                          </span>
                        </h3>
                      </div>
                      <hr
                        class="MuiDivider-root MuiDivider-fullWidth MuiDivider-flexItem css-yk10cl-MuiDivider-root"
                      />
                      <div
                        class="css-0"
                      >
                        <div
                          class="css-1fe9i7b"
                        >
                          Video Game
                        </div>
                        <h3
                          class="css-8kn4zf"
                        >
                          <span
                            class="css-pt399b"
                          >
                            transformers
                          </span>
                        </h3>
                        <div
                          class="css-1gqgroh"
                        >
                          released: 2003-01-01T05:00:00.000Z
                        </div>
                      </div>
                      <hr
                        class="MuiDivider-root MuiDivider-fullWidth MuiDivider-flexItem css-yk10cl-MuiDivider-root"
                      />
                      <div
                        class="css-0"
                      >
                        <div
                          class="css-1fe9i7b"
                        >
                          Movie
                        </div>
                        <h3
                          class="css-8kn4zf"
                        >
                          <span
                            class="css-pt399b"
                          >
                            transformers
                          </span>
                        </h3>
                      </div>
                      <hr
                        class="MuiDivider-root MuiDivider-fullWidth MuiDivider-flexItem css-yk10cl-MuiDivider-root"
                      />
                      <div
                        class="css-0"
                      >
                        <div
                          class="css-1fe9i7b"
                        >
                          Video Game
                        </div>
                        <h3
                          class="css-8kn4zf"
                        >
                          <span
                            class="css-pt399b"
                          >
                            transformers animated: the game
                          </span>
                        </h3>
                        <div
                          class="css-1gqgroh"
                        >
                          released: 2008-01-01T05:00:00.000Z
                        </div>
                      </div>
                      <hr
                        class="MuiDivider-root MuiDivider-fullWidth MuiDivider-flexItem css-yk10cl-MuiDivider-root"
                      />
                      <div
                        class="css-0"
                      >
                        <div
                          class="css-1fe9i7b"
                        >
                          TV Show
                        </div>
                        <h3
                          class="css-8kn4zf"
                        >
                          <span
                            class="css-pt399b"
                          >
                            transformers ascension
                          </span>
                        </h3>
                        <div
                          class="css-1gqgroh"
                        >
                          released: 2016-01-01T05:00:00.000Z
                        </div>
                      </div>
                      <hr
                        class="MuiDivider-root MuiDivider-fullWidth MuiDivider-flexItem css-yk10cl-MuiDivider-root"
                      />
                      <div
                        class="css-0"
                      >
                        <div
                          class="css-1fe9i7b"
                        >
                          TV Show
                        </div>
                        <h3
                          class="css-8kn4zf"
                        >
                          <span
                            class="css-pt399b"
                          >
                            transformers australia: grandpa unboxing
                          </span>
                        </h3>
                        <div
                          class="css-1gqgroh"
                        >
                          released: 2016-01-01T05:00:00.000Z
                        </div>
                      </div>
                      <hr
                        class="MuiDivider-root MuiDivider-fullWidth MuiDivider-flexItem css-yk10cl-MuiDivider-root"
                      />
                      <div
                        class="css-0"
                      >
                        <div
                          class="css-1fe9i7b"
                        >
                          Movie
                        </div>
                        <h3
                          class="css-8kn4zf"
                        >
                          <span
                            class="css-pt399b"
                          >
                            transformers cataclysm
                          </span>
                        </h3>
                      </div>
                      <hr
                        class="MuiDivider-root MuiDivider-fullWidth MuiDivider-flexItem css-yk10cl-MuiDivider-root"
                      />
                      <div
                        class="css-0"
                      >
                        <div
                          class="css-1fe9i7b"
                        >
                          TV Show
                        </div>
                        <h3
                          class="css-8kn4zf"
                        >
                          <span
                            class="css-pt399b"
                          >
                            transformers contention
                          </span>
                        </h3>
                        <div
                          class="css-1gqgroh"
                        >
                          released: 2018-01-01T05:00:00.000Z
                        </div>
                      </div>
                      <hr
                        class="MuiDivider-root MuiDivider-fullWidth MuiDivider-flexItem css-yk10cl-MuiDivider-root"
                      />
                      <div
                        class="css-0"
                      >
                        <div
                          class="css-1fe9i7b"
                        >
                          Movie
                        </div>
                        <h3
                          class="css-8kn4zf"
                        >
                          <span
                            class="css-pt399b"
                          >
                            transformers crusade
                          </span>
                        </h3>
                      </div>
                    </div>
                    <div
                      class="css-1clyo4s-MuiStack-root"
                    >
                      <button
                        class="MuiButton-root MuiButton-outlined MuiButton-outlinedPrimary MuiButton-sizeMedium MuiButton-outlinedSizeMedium MuiButtonBase-root Mui-disabled css-1rwt2y5-MuiButtonBase-root-MuiButton-root"
                        disabled=""
                        tabindex="-1"
                        type="button"
                      >
                        Back
                      </button>
                      <button
                        class="MuiButton-root MuiButton-outlined MuiButton-outlinedPrimary MuiButton-sizeMedium MuiButton-outlinedSizeMedium MuiButtonBase-root css-1rwt2y5-MuiButtonBase-root-MuiButton-root"
                        tabindex="0"
                        type="button"
                      >
                        Next
                        <span
                          class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
                        />
                      </button>
                    </div>
                  </div>
                </div>
              </div>
            </div>
          </div>
        </div>
        <div
          class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-12 MuiGrid-grid-md-3"
        >
          <div
            style="padding-left: 1rem; padding-right: 0.5rem;"
          >
            <div
              class="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation1 css-1ps6pg7-MuiPaper-root"
            >
              <div
                class="css-vcmx0t"
              >
                <div
                  class="css-365mzx"
                >
                  Stories
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;
